tests/beaker_tests/bgpneighboraf/test_bgpneighboraf.rb

```diff
@@ -113,10 +113,6 @@
   :manifest_props => "
     allowas_in                     => 'default',
     allowas_in_max                 => 'default',
-    cap_add_paths_receive          => 'default',
-    cap_add_paths_receive_disable  => 'default',
-    cap_add_paths_send             => 'default',
-    cap_add_paths_send_disable     => 'default',
```

Comment:
This was discussed during one of our PL's; ie. some properties (tri-states) shouldn't support explicit default. If you recall I was overruled on this one. :-)  
If you feel strongly about these particular properties then I'll rework the code to support it.